sprokit/processes/core/video_input_process.cxx

```diff
@@ -57,9 +57,13 @@ namespace kwiver {
 
 //                 (config-key, value-type, default-value, description )
 create_config_trait( video_filename, std::string, "", "Name of video file." );
-create_config_trait( frame_time, double, "0.3333333", "Inter frame time in seconds" );
+create_config_trait( frame_time, double, "0.3333333",
```

Comment:
Is this supposed to be 0.03333333?  I'm only asking because 30Hz is a common video rate while 3Hz is much less common.